dotnet/private/tests/nuget_structure/common.bzl

```diff
@@ -1,37 +1,40 @@
-"NuGet structure tests"
+"Common functionality for nuget structure tests"
 
 load("@bazel_skylib//lib:unittest.bzl", "analysistest", "asserts")
 
 # buildifier: disable=bzl-visibility
-load("//dotnet/private:providers.bzl", "DotnetAssemblyInfo")
+load("//dotnet/private:providers.bzl", "DotnetAssemblyInfo", "NuGetInfo")
+
+# buildifier: disable=bzl-visibility
+load("//dotnet/private/transitions:tfm_transition.bzl", "tfm_transition")
+
+# buildifier: disable=bzl-visibility
+load("//dotnet/private:common.bzl", "get_nuget_relative_path")
 
 def _get_nuget_relative_paths(files):
-    # The path of the files is of the form external/<packagename>.v<version>/<path within nuget package>
-    # So we remove the first two parts of the path to get the path within the nuget package.
-    nuget_paths_parts = [file.path.split("/")[2:] for file in files]
-    return ["/".join(path_parts) for path_parts in nuget_paths_parts]
+    return [get_nuget_relative_path(file) for file in files]
 
 def _nuget_structure_test_impl(ctx):
     env = analysistest.begin(ctx)
 
     target_under_test = analysistest.target_under_test(env)
     provider = target_under_test[DotnetAssemblyInfo]
 
-    libs_files = _get_nuget_relative_paths(provider.libs)
+    libs_files = _get_nuget_relative_paths(provider.lib)
     asserts.true(
         env,
         sorted(ctx.attr.expected_libs) == sorted(libs_files),
         "\nExpected libs:\n{}\nActual libs:\n{}".format(ctx.attr.expected_libs, libs_files),
     )
 
-    prefs_files = _get_nuget_relative_paths(provider.prefs)
+    prefs_files = _get_nuget_relative_paths(provider.ref)
     asserts.true(
         env,
         sorted(ctx.attr.expected_refs) == sorted(prefs_files),
         "\nExpected prefs:\n{}\nActual prefs:\n{}".format(ctx.attr.expected_refs, prefs_files),
     )
 
-    irefs_files = _get_nuget_relative_paths(provider.irefs)
+    irefs_files = _get_nuget_relative_paths(provider.iref)
```

Comment:
Not sure why these renames were needed or make sense.

Reply:
Yeah, I originally did them because they reflect the folders in the nuget packages. But I'll change it back I think.